met-web/src/components/engagement/view/widgets/Events/InPersonEvent.tsx

```diff
@@ -17,34 +17,34 @@ const InPersonEvent = ({ eventItem }: EventProps) => {
                 <MetBody>{eventItem.description}</MetBody>
             </Grid>
             <Grid container justifyContent={justifyContent} item xs={12}>
-                <Grid item xs={3}>
+                <Grid item xs={3} marginRight={2}>
```

Comment:
Let's meet to have a discussion on this. Just worried about using grid spacing and margin together. One of them should work